spec/api/coffee_shops_endpoint_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+require 'rack/test'
+
+RSpec.describe Api::V1::CoffeeShopsRoute do
+  include Rack::Test::Methods
+
+  def app
+    Api::V1::CoffeeShopsRoute
+  end
+
+  let(:csv_url) { 'https://raw.githubusercontent.com/Agilefreaks/test_oop/master/coffee_shops.csv' }
+  let(:user_coords) { { x: 47.6, y: -122.4 } }
+  let(:expected_response) do
+    [
+      { distance: 134.7088, name: 'Starbucks Rio De Janeiro' },
+      { distance: 136.2776, name: 'Starbucks Sydney' },
+      { distance: 178.4328, name: 'Starbucks Moscow' }
+    ]
+  end
+
+  before do
+    WebMock.disable_net_connect!(allow: 'raw.githubusercontent.com')
```

Comment:
the test will break as soon as the remote csv source changes

Reply:
Yeah, it will be better to use vcr and store the request and use the url from the ENV to disable web_mock in that context.